app/views/events/archive.html.haml

```diff
@@ -1,3 +1,7 @@
 - set_title t("events.archive.title")
+= render 'search_form'
```

Comment:
This search form probably can't search archived events.
I believe the search implementation in events.rb only shows upcoming events.